operators/src/engine/clonable_operator.rs

```diff
@@ -64,25 +63,17 @@ impl Clone for Box<dyn PlotOperator> {
     }
 }
 
-/// Helper trait for making boxed `InitializedOperator`s cloneable
-pub trait CloneableInitializedOperator {
-    type Descriptor: ResultDescriptor;
-    type Processor: QueryProcessor;
-
-    fn clone_boxed(&self) -> Box<dyn InitializedOperator<Self::Descriptor, Self::Processor>>;
-}
-
 /// Helper trait for making boxed `InitializedRasterOperator`s cloneable
 pub trait CloneableInitializedRasterOperator {
-    fn clone_boxed_raster(&self) -> Box<InitializedRasterOperator>;
+    fn clone_boxed_raster(&self) -> Box<dyn InitializedRasterOperator>;
```

Comment:
Why wasn't this `dyn` before and what changed?

Reply:
it was always dyn but hidden inside the type alias